src/fields.jl

```diff
@@ -98,6 +96,11 @@ function FaceFieldZ(::GPU, g::RegularCartesianGrid{T, <:AbstractRange}) where T
     FaceFieldZ{typeof(data), typeof(g)}(data, g)
 end
 
+CellField(g::Grid) = CellField(CPU(), g)
```

Comment:
These are useful defaults for working on the CPU.

Reply:
I don't know if it's good to assume that the default will be the CPU as we never run the model on CPUs lol, but I agree it makes sense from a code perspective.

Reply:
This default is useful for simple examples and post-processing. For the construction of `Model` the array type is always explicitly declared and this default is not used. We are building an abstraction system for calculus operators anyways for online calcs and flexible equation specification, so might as well make it as useful as possible for manipulating output. 

In the future, when we have an abstraction system for equations and time-stepping and perhaps have exotic column models and such implemented (via @johncmarshall54's comment in #174), we will run on CPUs more.

Reply:
Ah ok yeah that makes more sense then. We get to simplify the examples (and make the abstractions more usable) while the backend is always explicit so is left unaffected.